ctapipe/io/tableloader.py

```diff
@@ -26,6 +26,7 @@
 TRUE_IMAGES_GROUP = "/simulation/event/telescope/images"
 TRUE_PARAMETERS_GROUP = "/simulation/event/telescope/parameters"
 TRUE_IMPACT_GROUP = "/simulation/event/telescope/impact"
+SERVICE_GROUP = "/simulation/event/telescope/service"
```

Comment:
Service, by definition, is not per event. So this should be `/simulation/service/telescope`

Reply:
or `/service/simulation/telescope`? I am not sure why the configuration is in `/configuration/simulation/run` and not in ``simulation/configuration/run`.

@kosack do we rules from the top level data model for this hierarchy?

Reply:
Yes, `/simulation/service` is correct. There is not top-level `service` package in the data model. 

However, what is considered service vs configuration is not so well defined... Particularly for simulations where the border between what is a configuration parameter and what is a piece of data is somewhat blurry.  E.g. is a QE curve part of the configuration? (probably), but is the per-pixel QE value then also configuration or output? Not sure - I agree here it may be service info, as it is not required to drive the simulation, it's more an output.

Perhaps let's look at it that way:
- things that are part of what the simulation producer would include when launching the sims are in `/configuration/simulation`
- things that are derived during the simulation are in `/simulation/service/`

Therefore this data (e.g. per pixel QE curves) is service data, since it is generaetd from the input QE curves (as a function of wavelength) convolved with the wavelength response of each PMT inside the simulation code.